vita3k/gui/src/app_context_menu.cpp

```diff
@@ -71,9 +68,9 @@ static bool get_update_history(GuiState &gui, EmuEnvState &emuenv, const std::st
         while (update.second.find("<li>") != std::string::npos)
             if (update.second.find("<li>") != std::string::npos)
                 update.second.replace(update.second.find("<li>"), 4, reinterpret_cast<const char *>(u8"\u30FB")); // 00B7 or 2022 or 30FB or FF65
-        while (update.second.find(startpos) != std::string::npos)
+        while (update.second.find("<") != std::string::npos)
             if (update.second.find(">") + 1 != std::string::npos)
-                update.second.erase(update.second.find(startpos), update.second.find(endpos) + 1 - update.second.find(startpos));
+                update.second.erase(update.second.find("<"), update.second.find(">") + 1 - update.second.find("<"));
```

Comment:
```suggestion
                update.second.erase(update.second.find('<'), update.second.find('>') + 1 - update.second.find('<'));
```

Reply:
I did not know there was a difference between them. Thank you.